source/transactions/transactions.rst

```diff
@@ -12,7 +12,7 @@ Driver Transactions Specification
 :Status: Accepted (Could be Draft, Accepted, Rejected, Final, or Replaced)
 :Type: Standards
 :Minimum Server Version: 4.0 (The minimum server version this spec applies to)
-:Last Modified: 19-February-2019
+:Last Modified: 24-April-2019
```

Comment:
You might want to bump all of these dates before merging. No worries if not.

Reply:
Done.